src/Test/Plutip/Internal/BotPlutusInterface/Run.hs

```diff
@@ -132,5 +152,9 @@ runContract cEnv bpiWallet contract = do
 
       endState <- liftIO (readTVarIO $ ceContractState contractEnv)
       stats <- liftIO (readTVarIO $ ceContractStats contractEnv)
-      logs <- liftIO (readTVarIO $ ceContractLogs contractEnv)
+      logs <-
+        -- `bpi` collects logs in reversed order adding entries to the head of the `List`
+        -- we need them user-friendly-reversed
+        Bpi.LogsList . reverse . Bpi.getLogsList
```

Comment:
@zmrocze I also decided to move logs reversing to here, as when working with plutip from repl user may want to get logs from `ExecutionResult `

Reply:
makes a lot of sense, yes